packages/synapse-react-client/src/components/widgets/facet-nav/FacetNav.tsx

```diff
@@ -63,48 +69,30 @@ export function getFacets(
   return result
 }
 
-const FacetNav: React.FunctionComponent<FacetNavProps> = ({
-  facetsToPlot,
-}: FacetNavProps): JSX.Element => {
+const FacetNav: React.FunctionComponent<FacetNavProps> = (
```

Comment:
We should be able to just delete this file, making sure that

1. We move any constants or functions that are exported and used elsewhere
2. We migrate existing tests to use `PlotsContainer`